client/clientservice/src/main.cpp

```diff
@@ -53,6 +53,7 @@ po::variables_map parseCmdLine(int argc, char** argv) {
     ("tr-id", po::value<std::string>()->required(), "ID used to subscribe to replicas for data/hashes")
     ("tr-insecure", po::value<bool>()->default_value(false), "Testing only: Allow insecure connection with TRS on replicas")
     ("tr-tls-path", po::value<std::string>()->default_value(""), "Path to thin replica TLS certificates")
+    ("secrets-url", po::value<std::string>()->default_value(""), "URL to decrypt private keys")
```

Comment:
Is it used for private keys only? Or is it more generic "configuration"?

Reply:
It is only used for decrypting private keys.